STEPS.md

```
@@ -54,6 +55,44 @@
 
 ---
 
+## A11yTrait
+
+[Source](src/A11yTrait.php), [Example](tests/behat/features/a_11y.feature)
+
+>  Assess accessibility of rendered pages with axe-core.
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)